greenlight/lib/api/keys.py

```diff
@@ -14,6 +14,7 @@ def __init__(self, marionette_getter):
 
     @property
     def ACCEL(self):
-        if self.marionette_getter().session_capabilities['platformName'] == 'DARWIN':
+        if self.marionette_getter().session_capabilities['platformName'] == \
+                'DARWIN':
```

Comment:
I'm not using a linter, pep8 has a note about it in the "Maximum Line Length" section.